tests/acceptance/request_test.py

```diff
@@ -29,8 +31,14 @@ def test_app(request, **overrides):
 def validation_context(request, response=None):
     try:
         yield
-    except Exception:
+    except (
+        exceptions.RequestValidationError,
+        exceptions.ResponseValidationError,
+        exceptions.PathNotFoundError,
+    ):
         raise exception_response(206)
+    except Exception:
+        raise exception_response(400)
 
 
 validation_ctx_path = 'tests.acceptance.request_test.validation_context'
```

Comment:
This change covers it! There was a test before that should have been failing, but because it was catching all exceptions and not just specified ones it was passing.